### PR TITLE
Add MultiDB support for database:* commands

### LINKED ISSUE #589: firebase database:get on non-default database
Hi, 

My project has multiple databases (besides the default one)... how can I get the contents of another database? 

This retrieves the data from the default database:
    # firebase database:get /
{"test":123}

How can I get data from let's say db01?

### PR DESCRIPTION
**Pull request by rockwotj**
This change allows a `--instance` option that lets you change the
instance you're using. I went with `--instance` because it makes the
rest of the code a no-op change and fits in well :)

This also does validation against the databases for your project,
recommending others if an invalid one is selected.

Fixes #589

### CODE REVIEW

**Comment by rockwotj** [comment]
Example usage:

```
$ ./bin/firebase database:get --instance 'rockwood-test' /
{"external-ping":1511813046582,"internal-ping":1511813047263}
$ ./bin/firebase database:get --instance 'rockwood-test-896a0' /
null
$ ./bin/firebase database:get --instance 'rockwood-test-hkj' /
{"foo":{"bar":1508451921926},"parent":{"child1":"value1","child2":"value2"}}

$ ./bin/firebase database:get --instance 'not-a-database' /

Error: No instance named "not-a-database" found for project. Available options: [rockwood-test, rockwood-test-896a0, rockwood-test-fancy, rockwood-test-hkj] Please try a different project.

Having trouble? Try firebase database:get --help    
```

File: lib/getInstanceId.js [hunk 1]
```diff
@@ -12,11 +12,31 @@ var FirebaseError = require("./error");
...
+      return project.instances.database[0];
+    } else if (_.includes(project.instances.database, options.instance)) {
+      return options.instance;
+    } else {
+      throw new FirebaseError(
+        "No instance named '" +
+          options.instance +
+          "' found for project. Available options: [" +
+          project.instances.database.join(", ") +
+          "] Please try a different project.",
```
Comment (laurenzlong) [inline lib/getInstanceId.js]: Revise last sentence to: "Please choose a different instance on this project, or switch to a different project using `firebase use`"

File: lib/getInstanceId.js [hunk 2]
```diff
@@ -12,11 +12,31 @@ var FirebaseError = require("./error");
...
+    }
+    if (!_.has(options, ["instance"])) {
+      return project.instances.database[0];
+    } else if (_.includes(project.instances.database, options.instance)) {
+      return options.instance;
+    } else {
+      throw new FirebaseError(
+        "No instance named '" +
+          options.instance +
+          "' found for project. Available options: [" +
```
Comment (laurenzlong) [inline lib/getInstanceId.js]: Get rid of the square brackets, it looks more understandable as a list. 

```
Available options are: instanceA, instanceB.
```

**Comment by rockwotj** [comment]
Do you have a suggestion for the text? It's supposed to be an instance name without the firebaseio.com

I took a stab at it, PTAL

**Comment by laurenzlong** [comment]
How about:

use the database "<instance>@firebaseio.com" (if omitted, use default database instance)

**Comment by laurenzlong** [comment]
Approved pending API approval.

**Comment by rockwotj** [comment]
We're approved!

**Pull request merged**